protocol/bin/flow.js

```diff
@@ -383,18 +388,21 @@ function parseVariant(t, project) {
   }
 
   var type = parts.shift()
-  return t.cases
+  const cases = t.cases
     .map(c => {
       if (c.label.def) {
-        const bodyStr = c.body ? `, 'default': ?${c.body}` : ''
-        return `{ ${t.switch.name}: any${bodyStr} }`
+        return null
```

Comment:
what's the reason for returning null here?

Reply:
we have only once instance of this and its actually returning a void type so its just not printing anything. we can't return any like before since this is strict